### ISSUE TITLE
Consider moving NFS validation check to CAM registry pod

### ISSUE THREAD

**Issue by dymurray** [konveyor/mig-controller#532]
Today we are using the Velero pod to run an NFS validation check on `move` volumes. This has worked so far since we are managing the Velero builds ourselves we stick `nc` into the container to do the connection test.

This isn't ideal since we intend to eventually move to a Velero that is provided by a managed operator which shouldn't include CAM specific dependencies. Since an open migplan will lead to a registry pod on both source+destination, we can use this pod for the connection test.

**Comment by sseago** [comment]
Alternatively, we may want to consider a dedicated (minimal) container definition which we could use to just run the commands in (launched and exited as needed) rather than using the long-running registry pod.

**Comment by djwhatle** [comment]
@dymurray @sseago is this still a concern? 

Registry pods are not long-running after #717 so this check would need to go somewhere else.

**Comment by sseago** [comment]
It's probably still a concern, but it remains a low priority, and we obviously need a different solution now. The basic issue is we're overloading the velero pod with non-velero functionality because we need a convenient place to stuff "something we want to be able to run on a destination cluster". If there were more utility-type functions we needed to run in various places, making a dedicated image for it might make sense, but right now with the change around migration registry, there's no obvious alternative.
